src/Elcodi/Store/GeoBundle/Form/LocationSelectorBuilder.php

```diff
@@ -19,7 +19,7 @@
 
 use Doctrine\ORM\EntityNotFoundException;
 
-use Elcodi\Component\Geo\Services\Interfaces\LocationProviderInterface;
```

Comment:
Nopes. 

Our position about BC is that, during Beta stage, we'll comunicate all BCs as much as possible. We want our first release as clean as possible :)